horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m

```diff
@@ -283,11 +287,22 @@ function saveobj(~)
         function offset = get.offset(obj)
             offset = obj.offset_;
         end
+
+        function has = get.has_open_file_handle(obj)
+            has = ~isempty(obj.file_handle_);
+        end
     end
 
     %======================================================================
     % File handling/migration
     methods
+        function obj = ready_dump(obj)
```

Comment:
not at all clear why it's this name. If you can't find a better one, maybe comments to explain it